### PR TITLE
feat(account): kyc digital process revamp lot 2

### PR DESCRIPTION
**Pull request by MaximeBajeux**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/kyc-digital-process`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-14712
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/account/identity-documents/components/identity-documents-upload-detail/controller.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+export default class AccountUserIdentityDocumentsUploadDetailController {
+  /* @ngInject */
+  constructor() {
+    this.PROOF_TYPE = PROOF_TYPE;
+    this.PAGE_TYPE = PAGE_TYPE;
+    this.MAX_SIZE = MAX_SIZE;
+    this.multipleFile = false;
+    this.isValid = false;
+    this.badFileExtensionsFormatedList = undefined;
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/account/identity-documents/components/identity-documents-upload-detail/controller.js:16]: **nitpick:** Use `null` instead. If it needs to be `undefined` then this line has no impact, so can be removed.

File: packages/manager/apps/dedicated/client/app/account/identity-documents/components/identity-documents-upload-detail/template.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
+<div class="upload-documents-details d-flex w-100 flex-column">
+    <h3
+        class="oui-header__title d-flex align-items-center"
+        data-ng-click="$ctrl.goBack()"
+    >
+        <span class="oui-icon oui-icon-chevron-left"></span
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/account/identity-documents/components/identity-documents-upload-detail/template.html:6]:
```suggestion
        <span class="oui-icon oui-icon-chevron-left" aria-hidden="true"></span
```

**Review by JacquesLarique — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/account/identity-documents/components/identity-documents-upload-detail/template.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    <div data-ng-if="!$ctrl.fileExtensionsValid">
+        <oui-message class="my-1" data-type="error">
+            <span
+                data-translate="user_account_identity_documents_selection_file_format_invalid"
+                data-translate-values="{
+        values: $ctrl.badFileExtensionsFormatedList
+      }"
+            ></span>
+        </oui-message>
+    </div>
```
[the remark this answers is not in the document]
Comment (MaximeBajeux) [reply packages/manager/apps/dedicated/client/app/account/identity-documents/components/identity-documents-upload-detail/template.html:83]: I just kept that error banner in waiting of the update of ui-kit (with the file extension error handled)

File: packages/manager/apps/dedicated/client/app/account/identity-documents/components/identity-documents-upload-detail/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -0,0 +1,35 @@
+{
+  "upload-documents-details-title-identity": "Ajouter votre preuve d'identité",
+  "upload-documents-details-title-address": "Ajouter votre preuve d'adresse",
+  "upload-documents-details-title-aadhaar_card": "Ajouter votre Carte AADHAAR",
+  "upload-documents-details-title-authority_declaration": "Ajouter votre attestation sur l'honneur",
+  "upload-documents-details-title-vat": "Ajouter votre Carte preuve de TVA",
```
[the remark this answers is not in the document]
Comment (MaximeBajeux) [reply packages/manager/apps/dedicated/client/app/account/identity-documents/components/identity-documents-upload-detail/translations/Messages_fr_FR.json:6]: I don't know, but I copied it from the specs

File: packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.html [hunk 5]
```diff
@@ -1,166 +1,170 @@
...
+                <p
+                    data-translate="user_account_identity_documents_submit_success_message_description"
+                ></p>
+            </oui-message>
+            <!-- go back to dashboard option -->
+            <a
+                class="oui-link_icon mt-3"
+                data-ng-href="{{:: $ctrl.dashboardRedirectURL }}"
+                data-track-on="click"
+                data-track-name="{{:: $ctrl.TRACKING_TASK_TAG.gotToHub }}"
```
[the remark this answers is not in the document]
Comment (MaximeBajeux) [reply packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.html:116]: It was also with got in the constant file but I'm correcting that on both sides to be more semantically accurate.

File: packages/manager/apps/dedicated/client/app/account/identity-documents/components/identity-documents-upload-detail/controller.js [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+  clickAddDocuments() {
+    this.addDocuments({
+      proofType: this.proofType,
+      documentType: this.currentDocument,
+      files: this.files.flatMap((file) => file),
+    });
+    this.goBack();
+  }
+
+  checkValidity() {
```
[the remark this answers is not in the document]
Comment (MaximeBajeux) [reply packages/manager/apps/dedicated/client/app/account/identity-documents/components/identity-documents-upload-detail/controller.js:59]: this.isFileExtensionsValid should be removed after the update of ui-kit, I'm just refactoring the for loop to integrate it into the reduce.

File: packages/manager/apps/dedicated/client/app/account/identity-documents/components/identity-documents-upload-detail/template.html [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+            <ul>
+                <li
+                    data-ng-repeat="(index, acceptanceCriteria) in $ctrl.currentAcceptanceCriteria"
+                    data-translate="{{'upload-documents-details-acceptance-criteria-' + acceptanceCriteria }}"
+                    id="{{acceptanceCriteria + index}}"
+                ></li>
+            </ul>
+        </div>
+        <div
+            data-ng-class="'row row-cols-xs-1 row-cols-md-' + $ctrl.files.length + ' my-3'"
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/account/identity-documents/components/identity-documents-upload-detail/template.html:45]: **question:** Where are the class definitions for `row-cols-md-*` and `row-cols-xs-1`?

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/apps/dedicated/client/app/account/identity-documents/components/identity-documents-upload-detail/template.html (hunk 7 above)
Comment (MaximeBajeux) [reply packages/manager/apps/dedicated/client/app/account/identity-documents/components/identity-documents-upload-detail/template.html:45]: Them are from bootstrap

**Review by anooparveti — APPROVED** [review]

**Pull request merged**